pandas/core/internals.py

```diff
@@ -76,7 +78,169 @@
 from pandas.compat import range, map, zip, u
 
 
-class Block(PandasObject):
+class BlockOpsMixin(object):
```

Comment:
It looks like there is a `core.internal_ops` that is empty.  Was some of this intended to go in there?  I very much like the idea of having blocks ("vectors"?) implement their arithmetic methods, so Series/DataFrame can just pass through to those implementations.  shoyer opened an issue to that effect IIRC.

Reply:
Ah, these were temporarily in `internal_ops` and shared between a couple files, but then I made a refactor to remove internal_ops.